File: src/core/Utility.ts
export const toTimeString = (totalSeconds: number) => {
    const totalMs = totalSeconds * 1000;
    const result = new Date(totalMs).toISOString().slice(11, 19);

    return result;
};
export const getUnixTimestamp = () => Math.round(Date.now() / 1000);
export const sleep = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));

File: lib/core/utility.dart
String toTimeString(int totalSeconds) {
  final totalMs = totalSeconds * 1000;
  final duration = Duration(milliseconds: totalMs);
  return duration.toString().split('.').first.padLeft(8, '0');
}

int getUnixTimestamp() {
  return (DateTime.now().millisecondsSinceEpoch / 1000).round();
}

Future<void> sleep(int ms) {
  return Future.delayed(Duration(milliseconds: ms));
}
